DataBase/mongoQueryTimes.js

```diff
@@ -0,0 +1,28 @@
+use products
```

Comment:
curious what these lines are for

Reply:
good question! this file is to be called within the terminal in conjuncture with opening mongo.  The first line instructs mongo to use the correct database and the second has mongo display the available collections within the products db inside mongo